doc/quickstart.md

```diff
@@ -0,0 +1,210 @@
+
+-   can be destroyed with the command del `x`.
+
+For more details about Python variables, see [this](https://www.w3schools.com/python/python_variables.asp).
+
+ ## Static typed languages
+	
+A language is statically-typed if the type of a variable is known at compile-time instead of at run-time. Common examples of statically-
+typed languages include Java, C, C++, FORTRAN, Pascal and Scala, on the other hand, in python the type of a variable is known at the run-time, that's why we need to collect the garbage in the generated code, and raise some warnings/errors for the conflicts that can occur between dynamically typed languages (python) and statically typed languages(C/Fortran). See [this](https://en.wikipedia.org/wiki/Type_system#:~:text=In%20programming%20languages%2C%20a%20type,%2C%20expressions%2C%20functions%20or%20modules.) and [this](https://android.jlelse.eu/magic-lies-here-statically-typed-vs-dynamically-typed-languages-d151c7f95e2b#:~:text=Static%20typed%20languages,%2C%20FORTRAN%2C%20Pascal%20and%20Scala.) for more details.
+
+ ## Installation (see [README](https://github.com/pyccel/pyccel/blob/master/README.rst) file)
+
+ ## Command line usage
+
+-   Open a terminal app, iterm or terminal for MacOs, terminal for Linux.
+
+-   After the installation, type `pyccel`, the usage should be shown, If this is the case then the installation has succeeded.
+
+-   Create a Python file that contains simple lines of code to see what will happen.
+    -   To create the file `touch file_name.py`.
+    -   Use your favorite text editor to fill the file with some lines of code or just type `echo 'print("hello, world!\n")' > file_name.py` for a quick test, `cat file_name.py` to make sure that your lines in the file.
+    -   To generate the C/Fortran code form your Python code, type `pyccel file_name.py` or `pyccel file_name.py --language fortran` to generate Fortran code, and `pyccel file_name.py --language c` to generate C code.
+    -   No problems ? you should discover `__pyccel__` the directory that contains your generated code and some other stuff.
+
+E.g (using `@types` decorator and a recursive function with a typed return), To specify the types of the function arguments and its return, we need to import the `@types` decorator from pyccel.decorators (as you can see in `file_name.py` first line) and then specify the types for each function using this format:
+-   for the decorator `@types('1stArgType', '2ndArgType', 'NthArgType', results='return_type')` to declare arrays `@types('1stArgType[:]', '2ndArgType[:,:]', 'NthArgType[dimensions]', results='return_type')`, the expression `[:]` means that the array has 1 dimension and 2 dimensions with `[:,:]` , So the number of dimensions of an array related with how many colons you put in the square brackets sparated with a comma, `arr[:,:,:]` means that the array `arr` has 3 dimensions and so on.
```

Comment:
I would like to suggest something a bit clearer:
```
The syntax for the decorator is: `@types('1stArgType', '2ndArgType', 'NthArgType', results='return_type')`
or to declare arrays: `@types('1stArgType[:]', '2ndArgType[:,:]', 'NthArgType[dimensions]', results='return_type')`
The expression `[:]` means that the array has 1 dimension. 2 dimensions would be specified with `[:,:]`. The number of dimensions of an array is equal to the number of comma-separated colons in the square brackets. So `arr[:,:,:]` means that the array `arr` has 3 dimensions and so on.
```

Reply:
well structured, thanks